### PR TITLE
Add some httpbin-like functionality to web component tester

### PR DESCRIPTION
**Pull request by rictic**
Enables iron-ajax tests to be standalone and not hit the third party httpbin.org service.

### CODE REVIEW

File: runner/webserver.js [hunk 1]
```diff
@@ -139,6 +140,9 @@ module.exports = function(wct) {
           log:     wct.emit.bind(wct, 'log:debug'),
         }));
 
+        app.use('/httpbin', httpbin.httpbin);
+        console.log('installed httpbin');
```
Comment (azakus) [inline runner/webserver.js]: Use `wct.emit('log:debug', 'installed httpbin')`

File: runner/httpbin.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  var port = 7777;
+  server.listen(port);
+  server.port = port;
+  serverDestroy(server);
+  cleankill.onInterrupt(function(done) {
+    server.destroy();
+    server.on('close', done);
+  });
+
+  console.log('Server running at http://localhost:7777/');
```
Comment (azakus) [inline runner/httpbin.js]: Use `wct.emit('log:info', 'Server running at http://localhost:7777/')`

**Comment by azakus** [comment]
LGTM with changes to use the wct logging system.

File: runner/httpbin.js (hunk 2 above)
Comment (azakus) [inline runner/httpbin.js]: Optional: use `portScanner` to find another sauce supported open port (`SAUCE_PORTS`)

File: runner/webserver.js (hunk 1 above)
Comment (rictic) [inline runner/webserver.js]: Just removed the line, I was trying to debug whether this line was being reached and whether `npm link` had worked like I expected it to.

File: runner/httpbin.js (hunk 2 above)
Comment (rictic) [inline runner/httpbin.js]: Used portScanner.

I think console.log is appropriate for this sort of debug server functionality, as it's going to be the only thing output, and it should always be actionable.

Comment (azakus) [inline runner/httpbin.js]: The only real difference is that `wct.emit('log:info')` will colorize the line in the same style as the line announcing the webserver's port. I think it's nice to keep them consistent.

Comment (rictic) [inline runner/httpbin.js]: Talked it out offline, it wouldn't be easy to pipe this little standalone server into the rest of the machinery.

**Pull request merged**